docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst

```diff
@@ -1,22 +1,26 @@
 .. _slurm-image-config:
 
-####################################
- Provide a Singularity Images Cache
-####################################
+#################################
+ Provide a Container Image Cache
+#################################
 
-When the cluster does not have Internet access or if you want to provide a local cache of
-Singularity images to be used on the cluster, you can put the Singularity images you want in the
-applicable directory under the ``singularity_image_root`` directory. If found, the image reference
-is translated to a local file path and substituted in the ``singularity run`` command to avoid the
-need for Singularity to download and convert the image for each user.
+When the cluster does not have Internet access or if you want to provide a local cache of container
+images to improve performance, you can download the desired container images to a shared directory
+and then reference them using file system paths instead of docker registry references.
 
-Each version of Determined utilizes specifically-tagged Docker containers. The image tags referenced
-by default in this version of Determined are described below.
+There are two mechanisms you can use to reference cached container images depending upon the
+container runtime in use.
+
+   -  :ref:`referencing-local-image-paths`
+   -  :ref:`singularity-image-cache`
 
 ***********************
  Default Docker Images
 ***********************
 
+Each version of Determined utilizes specifically-tagged Docker containers. The image tags referenced
+by default in this version of Determined are described below.
```

Comment:
Is there a process to keep this table up to date?

Reply:
Yes, there is a process where exact matches to determined container name  are updated to the new version in the docs.